src/main/webapp/app/shared/metis/posting-header/post-header/post-header.component.html

```diff
@@ -1,25 +1,39 @@
-<div class="row justify-content-between my-1">
-    <div class="col-auto pe-0">
-        <span class="posting-header header-author-date">
+<div class="d-flex justify-content-between my-1">
+    <div class="align-items-center">
+        <span class="fs-medium header-author-date">
             @if (posting.author) {
-                <span class="posting-author-role" ngbTooltip="{{ userAuthorityTooltip | artemisTranslate }}">
-                    <fa-icon [icon]="userAuthorityIcon" />
+                <span class="fs-6" ngbTooltip="{{ userAuthorityTooltip | artemisTranslate }}">
+                    <span id="role-badge" class="badge fs-small p-1 rounded-3" [class]="'badge-' + userAuthority"> </span>
                 </span>
-                <span class="posting-author">
+                <span class="fs-large pe-1 fw-bold">
                     {{ posting.author!.name }}
                 </span>
             }
+            <span class="px-1"> - </span>
             @if (postingIsOfToday) {
                 <span class="today-flag">{{ todayFlag | artemisTranslate }}</span>
             }
             <span [disableTooltip]="postingIsOfToday" ngbTooltip="{{ posting.creationDate | artemisDate: 'time' }}">
                 {{ postingIsOfToday ? (posting.creationDate | artemisDate: 'time') : (posting.creationDate | artemisDate: 'short-date') }}
             </span>
         </span>
+        @if (posting.resolved) {
+            <fa-icon
+                [icon]="faCheckSquare"
+                iconSize="xs"
+                class="col-auto pe-0 resolved px-1"
+                [ngbTooltip]="'artemisApp.metis.post.postMarkedAsResolvedTooltip' | artemisTranslate"
+            />
+        }
+        @if (isCommunicationPage && (!lastReadDate || (lastReadDate && posting.creationDate && posting.creationDate.isAfter(lastReadDate))) && !isAuthorOfPosting) {
+            <span class="badge bg-secondary hideAfter5Seconds">{{ 'global.generic.new' | artemisTranslate }}</span>
+        }
+    </div>
+    <div class="mx-2 my-1">
```

Comment:
Please make sure that these icons don't wrap 
<img width="345" alt="image" src="https://github.com/ls1intum/Artemis/assets/75392103/8b008906-e7b2-4f33-bca0-edf3b20fc65a">